themes/default/content/blog/stack-readme/index.md

```diff
@@ -0,0 +1,84 @@
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - evan-boyle
+    - devon-grove
+    - myles-haynes
+    - casey-huang
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+# See the blogging docs at https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md.
+# for additional details, and please remove these comments before submitting for review.
+---
+
+Starting today, users can create Stack READMEs in the [Pulumi Service](https://app.pulumi.com) that dynamically update based on [Stack Outputs](https://www.pulumi.com/learn/building-with-pulumi/stack-outputs). Stack READMEs interpolate output variables on the stack (${outputs.instances[0].ARN}) so that each stack can construct links to dashboards, shell commands, and other pieces of documentation. All of this content stays up to date as you stand up new stacks, rename resources, and refactor your infrastructure.
+
+<!--more-->
+
+There are a lot of operational activities that happen around [Pulumi Stacks](https://www.pulumi.com/docs/intro/concepts/stack), however, the information for these activities live across many places. Our [User Experience team](https://www.pulumi.com/blog/get-to-know-pulumis-ux-team) recently conducted user experience interviews with our customers and we learned users are moving across many web apps and local apps to manage their deployments- from their cloud Console, to the Pulumi Service User Interface (UI), to the CLI, and so on. By collecting resource outputs in the Pulumi Service UI, we are aiming to ease this friction and collect relevant Stack information in one place.
+
+README templates can reference Stack outputs and resource properties, such as `${outputs.foo}`. To walk you through a tangible example, a user can deploy an RDS instance and then use a variable in their README template to link to their CloudWatch dashboard to keep an eye on operational metrics. This CloudWatch dashboard link will dynamically update when deployments happen. You can use the same README template across dev, testing and production and have the correct dashboard links for each Stack.
+
+The new experience lives in the Stack page, which can be navigated to through Projects and clicking on the specific Stack you want to view the README for. Let's take a look at this new feature!
+
+![Stack READMEs in the Pulumi Console](stack-readme.png)
+
+To illustrate, let's look at the new Stack README for the Pulumi Service dev stack.
+
+```markdown
+# Pulumi Service README
+​
+[Sign in to AWS to view stack resources!](https://top-secret-url.com)
+​
+I am...
+​
+* [an on call engineer](#oncall)
+* [monitoring a deployment](#cicd)
+* [a new hire looking to onboard](#onboard)
+* [interested in querying for business insights](#insights)
+​
+-----
+​
+<a name="oncall"></a>
```

Comment:
I'm not sure if this is valid markdown (or if it actually works in our rendering engine) let's remove it.